### ISSUE TITLE
Example from readme does not work with 2.0.0

### ISSUE THREAD

**Issue by hydrogen18** [cequel/cequel#328]
Hello,

Using Cequel 2.0.0 I can no longer `include Cequel::Record`. I am using ruby 2.3.0

```
[ericu-destroyer-of-worlds] cequel_playground$ cat blog.rb
require 'cequel'
require 'cequel/version'
puts Cequel::VERSION

class Blog
  include Cequel::Record

  key :subdomain, :text
  column :name, :text
  column :description, :text
end
[ericu-destroyer-of-worlds] cequel_playground$ ruby ./blog.rb
2.0.0
/home/ericu/.rvm/gems/ruby-2.3.0@cequel/gems/cequel-2.0.0/lib/cequel/record.rb:97:in `block in <module:Record>': uninitialized constant ActiveModel::Serializers::Xml (NameError)
	from /home/ericu/.rvm/gems/ruby-2.3.0@cequel/gems/activesupport-5.0.0.1/lib/active_support/concern.rb:120:in `class_eval'
	from /home/ericu/.rvm/gems/ruby-2.3.0@cequel/gems/activesupport-5.0.0.1/lib/active_support/concern.rb:120:in `append_features'
	from ./blog.rb:6:in `include'
	from ./blog.rb:6:in `<class:Blog>'
	from ./blog.rb:5:in `<main>'
```

Here are the other gems in my gemset presently

```
[ericu-destroyer-of-worlds] cequel_playground$ gem install cequel
Fetching: i18n-0.7.0.gem (100%)
Successfully installed i18n-0.7.0
Fetching: thread_safe-0.3.5.gem (100%)
Successfully installed thread_safe-0.3.5
Fetching: tzinfo-1.2.2.gem (100%)
Successfully installed tzinfo-1.2.2
Fetching: concurrent-ruby-1.0.2.gem (100%)
Successfully installed concurrent-ruby-1.0.2
Fetching: activesupport-5.0.0.1.gem (100%)
Successfully installed activesupport-5.0.0.1
Fetching: activemodel-5.0.0.1.gem (100%)
Successfully installed activemodel-5.0.0.1
Fetching: ione-1.2.3.gem (100%)
Successfully installed ione-1.2.3
Fetching: cassandra-driver-3.0.3.gem (100%)
Building native extensions.  This could take a while...
Successfully installed cassandra-driver-3.0.3
Fetching: cequel-2.0.0.gem (100%)
Successfully installed cequel-2.0.0
Parsing documentation for i18n-0.7.0
Installing ri documentation for i18n-0.7.0
Parsing documentation for thread_safe-0.3.5
Installing ri documentation for thread_safe-0.3.5
Parsing documentation for tzinfo-1.2.2
Installing ri documentation for tzinfo-1.2.2
Parsing documentation for concurrent-ruby-1.0.2
Installing ri documentation for concurrent-ruby-1.0.2
Parsing documentation for activesupport-5.0.0.1
Installing ri documentation for activesupport-5.0.0.1
Parsing documentation for activemodel-5.0.0.1
Installing ri documentation for activemodel-5.0.0.1
Parsing documentation for ione-1.2.3
Installing ri documentation for ione-1.2.3
Parsing documentation for cassandra-driver-3.0.3
Installing ri documentation for cassandra-driver-3.0.3
Parsing documentation for cequel-2.0.0
Installing ri documentation for cequel-2.0.0
Done installing documentation for i18n, thread_safe, tzinfo, concurrent-ruby, activesupport, activemodel, ione, cassandra-driver, cequel after 11 seconds
9 gems installed
```

**Comment by pezra** [comment]
@hydrogen18, with rails 5 you need to add `gem 'activemodel-serializers-xml'` to your Gemfile. #329 removes the need to that but it has not been released yet.

**Comment by hydrogen18** [comment]
OK, subscribed to the other issue.

**Comment by pezra** [comment]
For the record, this is fixed in version 2.0.1
